src/tiramisu_expr.cpp

```diff
@@ -3,6 +3,11 @@
 
 namespace tiramisu
 {
+bool global::auto_data_mapping = false;
```

Comment:
Why did you move them here? I assume because the original order created a problem.

Would the code work if you move them to the header file core.h instead? These variables do not belong semantically to the file expr.h since they are general and not related to expressions only. So moving them somewhere else makes more sense.

Reply:
.

Reply:
> So moving them somewhere else makes more sense.

Yes, we can move them here.

I made additional testing of the library under memory debugger (Dr. Memory) and found some bugs which were leading to Visual Studio problems. They were all related to my ISL build.

Now both MinGW and Visual Studio build of Tiramisu works fine.